pkg/controller/changetierrequest/changetierrequest_controller.go

```diff
@@ -105,12 +108,22 @@ func (r *ReconcileChangeTierRequest) Reconcile(request reconcile.Request) (recon
 	}
 
 	reqLogger.Info("Change of the tier is completed")
+	err = r.setStatusChangeComplete(changeTierRequest)
+	if err != nil {
+		reqLogger.Error(err, "unable to set change complete status to ChangeTierRequest")
+		return reconcile.Result{}, err
+	}
+
 	return reconcile.Result{
 		Requeue:      true,
-		RequeueAfter: r.config.GetDurationBeforeChangeRequestDeletion(),
-	}, r.setStatusChangeComplete(changeTierRequest)
+		RequeueAfter: r.config.GetDurationBeforeNotificationDeletion(),
```

Comment:
it should be GetDurationBeforeChangeRequestDeletion, right?
```suggestion
		RequeueAfter: r.config.GetDurationBeforeChangeRequestDeletion(),
```

Reply:
good catch, fixed here: 7a45d050f1b4b9adfddabbbe31743d7a35cd725b